deflate.h

```diff
@@ -121,7 +121,7 @@ typedef struct internal_state {
 
                 /* used by deflate.c: */
 
-    unsigned int  w_size;            /* LZ77 window size (32K by default) */
+    uint16_t      w_size;            /* LZ77 window size (32K by default) */
```

Comment:
This change seems misplaced in this commit since it can affect other archs too.

Reply:
It's missing preprocessor test as when I made the change, it caused issues on other platforms even though none of the other platforms should have HAVE_PPC_ALTIVEC set. I can add the test back and retry with Travis.